cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_vcf.py

```diff
@@ -114,33 +120,59 @@ def _add_transcripts_to_vcf(self, vcf_variant_dict, transcript_dict):
                 tfx_c_dots.append(transcript.hgvs_c_dot)
                 tfx_p1.append(transcript.hgvs_p_dot_one)
                 tfx_p3.append(transcript.hgvs_p_dot_three)
-                tfx_splice.append(transcript.splicing)
                 tfx_refseq_transcripts.append(transcript.refseq_transcript)
                 tfx_variant_effects.append(transcript.variant_effect)
                 tfx_variant_types.append(transcript.variant_type)
                 tfx_protein_transcripts.append(transcript.protein_transcript)
                 tfx_amino_acid_positions.append(transcript.hgvs_amino_acid_position)
     
+            vcf_record.INFO[TranscriptEffect.TFX_SPLICE.value] = [tfx_splice]
+            vcf_record.INFO[TranscriptEffect.TFX_REFERENCE_CONTEXT.value] = [tfx_reference_context]
```

Comment:
As a matter of convention, it might make sense to reserve the `TFX_` prefix for only those INFO fields that use multiple values per transcript. This was the convention I had in mind when I originally introduced those names, and now that `TFX_SPLICING` and `TFX_REFERENCE_CONTEXT` are single-valued fields, they are just "regular" INFO fields like any other.

Of course, you could also make the argument that the prefix only indicates the data are added by the TxEff module, which still applies to these two fields.

I don't suppose I have a strong preference either way, but it can be nice to have a definitive way to determine which fields should be split and assigned by transcript and which shouldn't (such as if they start with TFX_ or not).

Reply:
It's fine to prefix all tx eff module fields with `TFX_`
I agree that adding SINGLE_VALUE and MULTI_VALUE to each symbol definition would be excessively verbose.
I think adding documentation to the VCF header and in CGD's VcfParserHelper are good ideas.

Reply:
In VCFParser helper the comments on the fields mention whether it is an array of values or a single value: 
```java
/** A parallel array of transcript effect ProteinVariant amino acid position */
	private static final String TFX_AMINO_ACID_POSITION = "TFX_AMINO_ACID_POSITION";

/** A single value that is the reference sequences surrounding a variant */
	private static final String TFX_REFERENCE_CONTEXT = "TFX_REFERENCE_CONTEXT";
```

I'm going to update the field definitions in the VCF.  What do you think looks better: 
* Ex: "Variant effect annotation (parallel array of values)" or "List of variant effect annotations"? 
* Ex: "Reference context (single value) or "Single value reference context"?

Reply:
I like the `Reference context (single value)` because it makes the type more distinct from the field description, and also makes it more apparent a separate description of the structure is present